examples/basic.py

```diff
@@ -15,22 +17,28 @@ def evaluate_metric(assignments):
 
   connection = Connection(client_token=the_args.client_token, _show_deprecation_warning=False)
 
+  
+  sigopt.log_dataset("Example dataset") #Descriptor of what kind of dataset you are modeling
+  sigopt.log_metadata(key="Dataset Source", value="Example Source") #Useful for keeping track of where you got the data 
+  sigopt.log_metadata(key="Feature Pipeline Name", value="Example Pipeline") #e.g. Sklern, xgboost, etc.
+  sigopt.log_model("Example Model Technique") # What kind of learning you are attemping
   # Create an experiment with one paramter, x
```

Comment:
consistent formatting: do we want inline comments or above-comments? and there should be a space between `#` and the first character (lingering todo item to implement linting / auto-formatting on this repo)